### PR TITLE
remove pretrack icon

### PR DESCRIPTION
**Pull request by yarneo**
removing pretrack icon due to final product review comments

### CODE REVIEW

File: assets/icons/pretrack.svg [hunk 1]
```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zakj) [inline assets/icons/pretrack.svg]: please either leave this icon in or remove references to it in the style guide

**Comment by yarneo** [comment]
thanks. not entirely sure how to remove the stuff from the compiled/ folders. im guessing these are automatically generated with a certain script?

**Comment by tdumitrescu** [comment]
not the style guide tho

**Comment by zakj** [comment]
shipped in 0.75.4

**Comment by yarneo** [comment]
thank you!

**Pull request closed without merging**